src/Robo/Tasks/Drush.php

```diff
@@ -0,0 +1,267 @@
+<?php
+
+namespace Acquia\Blt\Robo\Tasks;
+
+use Robo\Task\CommandStack;
+use Robo\Contract\VerbosityThresholdInterface;
+use Robo\Common\CommandArguments;
+
+/**
+ * Runs Drush commands in stack. You can use `stopOnFail()` to point that stack should be terminated on first fail.
+ *
+ * ``` php
+ * $this->taskDrush()
+ *     ->drush('updb')
+ *     ->drush('cr')
+ *     ->run();
+ * ```
+ */
+class Drush extends CommandStack
+{
+  use CommandArguments;
+
+  /**
+   * Site alias to prepend to each command.
+   *
+   * @var string
+   */
+  private $alias;
+
+  /**
+   * Directory to execute the command from.
+   *
+   * @var string
+   *
+   * @see ExecTrait::$workingDirectory
+   */
+  private $dir;
+
+  /**
+   * Site uri to append uri option to each command.
+   *
+   * @var string
+   */
+  private $uri;
+
+  /**
+   * Assume 'yes' or 'no' to all prompts.
+   *
+   * @var string|bool
+   */
+  private $assume;
+
+  /**
+   * Attach tty to process for interactive input.
+   *
+   * @var bool
+   *
+   * @see ExecTrait::$interactive
+   */
+  private $passthru;
```

Comment:
Happy to do that - only left it in as I was unsure if you were hoping to continue to use that terminology.